Lib/test/test_peg_generator/test_c_parser.py

```diff
@@ -391,6 +391,13 @@ def test_headers_and_trailer(self) -> None:
         self.assertTrue("SOME SUBHEADER" in parser_source)
         self.assertTrue("SOME TRAILER" in parser_source)
 
+    def test_multiple_lookahead(self) -> None:
+        grammar_source = 'start: !"(" NEWLINE+ !")"'
+        grammar = parse_string(grammar_source, GrammarParser)
+        parser_source = generate_c_parser_source(grammar)
+
+        self.assertNotIn("int None_1", parser_source)
```

Comment:
Hummmm. I don't like this test because is checking the implementation. What goes wrong exactly now when there are two negative lookaheads?

Reply:
> What goes wrong exactly now when there are two negative lookaheads?
```
gcc -pthread -c -Wno-unused-result -Wsign-compare -g -Og -Wall    -std=c99 -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-missing-field-initializers -Werror=implicit-function-declaration -fvisibility=hidden  -I./Include/internal  -I. -I./Include    -DPy_BUILD_CORE -o Parser/pegen/parse.o Parser/pegen/parse.c
Parser/pegen/parse.c: In function ‘invalid_import_from_targets_rule’:
Parser/pegen/parse.c:12249:13: warning: unused variable ‘None_1’ [-Wunused-variable]
12249 |         int None_1;
      |             ^~~~~~
```

Reply:
> Hummmm. I don't like this test because is checking the implementation.

I agree with you but this is an implementation problem so I dont know exactly how to write a better test for this

Reply:
> I agree with you but this is an implementation problem so I dont know exactly how to write a better test for this

Yeah, let's remove the test and let's focus on why mypy has not found this.

Reply:
CCallMakerVisitor's return value is Any

Reply:
:facepalm: 

Ok, will prepare a PR to fix mypy, I found a couple of other stuff.

Reply:
Opened https://github.com/python/cpython/pull/20297 to fix mypy issues